### PR TITLE
Covid 19 App: React capstone

### PR DESCRIPTION
**Pull request by Emmy-github-webdev**
Covid 19 App: React capstone

In this project I
- Use Covid 19 Data API
- Create two pages- The home page, and the detail page
- Ensure there is no linter error
- Follow Gitflow
- Follow best practices for HTML & CSS, and JavaScript
- Filter the data retrieve from the API using a stateless component 
- Create unit test for pure function in my Redux code
- Create integration tests for my application using the React Testing Library
- Add video presentation link

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import './App.css';
+import {
+  BrowserRouter as Router, Routes, Route,
+} from 'react-router-dom';
+import HomeScreen from './screens/HomeScreen';
+import VaccineDetailScreen from './screens/VaccineDetailScreen';
+
+function App() {
+  return (
```
Comment (rukundoeric) [inline src/App.js]: - [ ] <b>REQUIRED:<b/> Prefer using ES6 arrow function. Actually, there is nothing wrong with ES5 functions syntax but it is mandatory to use es6 features in the javascript module. besides, Arrow functions provide a simplified structure, allowing for a more concise way to write function expressions. which makes them better than ES5 functions(function declarations). Here is the syntax of an ES6 arrow function...
```javascript
  const functionName = () => {
     // code here
  }
```

File: src/screens/VaccineScreen.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        <BsArrowRightCircleFill color="white" />
+      </Link>
+      <h1>{vaccine.All.country}</h1>
+      <h1>{vaccine.All.people_vaccinated.toLocaleString()}</h1>
+    </div>
+  </>
+);
+
+VaccineScreen.propTypes = {
+  vaccine: PropTypes.object.isRequired,
```
Comment (rukundoeric) [inline src/screens/VaccineScreen.js]: - [ ] <b>REQUIRED:</b> You can fix the above `eslint` error by using the following line of code on line 21.

```js
VaccineScreen.propTypes = {
  vaccine: PropTypes.shape({ }).isRequired
  }
```

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @Emmy-github-webdev, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] Your app does not have a filter functionality.  the project requires that you must have some sort of a filter, maybe my category or something else. Kindly implement it to comply with project requirements.
  - [ ] You only tested `snapshots` but the project requires to have unit tests for several components of the app and some pure functions in Redux code. Kindly add more tests cases.
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/App.js (hunk 1 above)
Comment (Emmy-github-webdev) [reply src/App.js]: Thanks, @rukundoeric. Recommendation applied

File: src/screens/VaccineScreen.js (hunk 2 above)
Comment (Emmy-github-webdev) [reply src/screens/VaccineScreen.js]: Thanks, @rukundoeric. Recommendation applied

**Review by elyor-doniyorov — APPROVED** [review]
Hi @Emmy-github-webdev👋 , 

## To Highlight 👏 
- The Home page has list of categories that could be filtered using given parameters;
- Have unit tests for the app components;

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## STATUS APPROVED ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**